lib/screens_web/controllers/screen_api_controller.ex

```diff
@@ -26,10 +26,11 @@ defmodule ScreensWeb.ScreenApiController do
     end
   end
 
-  def show(conn, %{"id" => screen_id, "last_refresh" => last_refresh}) do
+  def show(conn, %{"id" => screen_id, "last_refresh" => last_refresh} = params) do
```

Comment:
Minor thing, but any particular reason to hold onto `params` here instead it pattern matching out `source` like the other two fields here?

Reply:
Mostly to avoid making a second function for cases where there is no source at all. URLs set for live screens and pulling up a screen in the browser will not have this `source` a majority of the time. Accessing the param using `params["source"]` allows me to conditionally add the source to the logs if it was supplied. If not, it looks for `is_real_screen=true`. If that's also not there, it skips logging.

Reply:
Ah okay. So I see that there's a Screenplay PR to add the source param. And you mentioned that real, rider-facing screens typically won't have the source param at all. Is that basically how we'll filter requests in Splunk?

Reply:
Correct. We want to give Ana a way to filter out simulation logs in her Splunk queries while also giving us a way to see data requests from Screenplay users. And because we don't want to have to change all of the live URLs, the current URLs with `is_real_screen=true` will be translated to `source=real_screen`.

Reply:
Make sense!